app/services/data_migrations/backfill_application_choices_with_work_experiences.rb

```diff
@@ -0,0 +1,34 @@
+module DataMigrations
+  class BackfillApplicationChoicesWithWorkExperiences
+    TIMESTAMP = 20240822105049
+    MANUAL_RUN = true
+
+    def change
+      time_now = Time.zone.now
+
+      choices_without_work_histories.each_slice(5000).with_index do |ids, index|
+        next_batch_time = time_now + index.minutes
+        MigrateApplicationChoicesWorker.perform_at(next_batch_time, ids)
+      end
+    end
+
+    def choices_without_work_histories
+      sql = <<-SQL
```

Comment:
I think we probably need `distinct` in here somewhere.

Reply:
`The UNION operator selects only distinct values by default. To allow duplicate values, use UNION ALL` according to this https://www.w3schools.com/sql/sql_union.asp

Reply:
I see! Thanks for explaing!